pytest/common_lib/contracts.py

```diff
@@ -16,8 +16,12 @@
     / "migration"
     / "migration_contract.wasm"
 )
-TESTNET_ACCOUNT_ID = "v1.signer-prod.testnet"
-MAINNET_ACCOUNT_ID = "v1.signer"
+CURRENT_MAINNET_CONTRACT_PATH = (
+    MPC_REPO_DIR / "pytest" / "common_lib" / "current_contracts" / "signer_mainnet.wasm"
```

Comment:
Why don't we rely on the `contract_history` crate? 

Having the `.wasm` binaries in the pytest folder means yet another two binaries to maintain (which, I thought should be handled by the `contract_history` crate).

Reply:
Yeah that's why I added the explicit export script. Wasn't too happy about this solution, so the pytest fixture variant you proposed is much better.

Updated in https://github.com/near/mpc/pull/1031/commits/6a5f763a68e5197e947891817039ffb670751f47